### PR TITLE
Test

### PR DESCRIPTION
**Pull request by HanqingZheng**

### CODE REVIEW

**Comment by HanqingZheng-ms** [comment]
Docs Build status updates of commit _[ae684a2](https://github.com/HanqingZheng/redirect-depot/commits/ae684a22a56dce48b4ab4b0fcf2498eb9c7b46c6)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[redirect-depot/index.md](https://github.com/HanqingZheng/redirect-depot/blob/test/redirect-depot/index.md) | :x:Error | | [Details](#user-content-5e3fb48cc109b958cfab867dc6d1d0a7-c42f91b3)

<a id="5e3fb48cc109b958cfab867dc6d1d0a7-c42f91b3"></a>
### [redirect-depot/index.md](https://github.com/HanqingZheng/redirect-depot/blob/test/redirect-depot/index.md)
 - **Line 8, Column 3**: **[Error: violate-schema]** `````Expected type String, please input String or type compatible with String.`````
 - **Line 43, Column 103**: **[Warning: file-not-found - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/file-not-found?branch=main)]** `````Invalid file link: '../code-quality/how-to-suppress-warnings-by-using-the-menu-item.md'.`````
 - **Line 68, Column 130**: **[Warning: file-not-found - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/file-not-found?branch=main)]** `````Invalid file link: '../code-quality/code-analysis-for-managed-code-warnings.md'.`````
 - **Line 106, Column 2**: **[Warning: codesnippet-not-found]** `````Invalid code snippet link: '../code-quality/codesnippet/CPP/in-source-suppression-overview_1.cpp'.`````
 - **Line 107, Column 2**: **[Warning: codesnippet-not-found]** `````Invalid code snippet link: '../code-quality/codesnippet/VisualBasic/in-source-suppression-overview_1.vb'.`````
 - **Line 108, Column 2**: **[Warning: codesnippet-not-found]** `````Invalid code snippet link: '../code-quality/codesnippet/CSharp/in-source-suppression-overview_1.cs'.`````

For more details, please refer to the [build report](https://buildapi.ppe.docs.microsoft.com/Output/PullRequest/269490f8-493a-1ac8-4f64-0d7ad8d44892/202209190619542694-21/BuildReport?accessString=9173ab986ee394c1496d444102d90030f862a41184f437441f897e537aab103a).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://docs-portal-pubdev-wus.azurewebsites.net/#/repos/269490f8-493a-1ac8-4f64-0d7ad8d44892?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings or suggestions unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Review by HanqingZheng-ms — APPROVED** [review]

**Review by HanqingZheng-ms — COMMENTED** [review]
????

File: redirect-depot/index.md [hunk 1]
```diff
@@ -107,32 +107,18 @@ CA_SUPPRESS_MESSAGE("Rule Category", "Rule Id", Justification = "Justification",
  [!code-vb[InSourceSuppression#1](../code-quality/codesnippet/VisualBasic/in-source-suppression-overview_1.vb)]
  [!code-cs[InSourceSuppression#1](../code-quality/codesnippet/CSharp/in-source-suppression-overview_1.cs)]  
   
-## Generated Code  
- Managed code compilers and some third-party tools generate code to facilitate rapid code development. Compiler-generated code that appears in source files is usually marked with the **GeneratedCodeAttribute** attribute.
```
Comment (HanqingZheng-ms) [inline redirect-depot/index.md:111]: asfsdfsdfasdf

File: redirect-depot/index.md [hunk 2]
```diff
@@ -107,32 +107,18 @@ CA_SUPPRESS_MESSAGE("Rule Category", "Rule Id", Justification = "Justification",
  [!code-vb[InSourceSuppression#1](../code-quality/codesnippet/VisualBasic/in-source-suppression-overview_1.vb)]
  [!code-cs[InSourceSuppression#1](../code-quality/codesnippet/CSharp/in-source-suppression-overview_1.cs)]  
   
-## Generated Code  
- Managed code compilers and some third-party tools generate code to facilitate rapid code development. Compiler-generated code that appears in source files is usually marked with the **GeneratedCodeAttribute** attribute.  
-  
- You can choose whether to suppress Code Analysis warnings and errors for generated code. For information about how to suppress such warnings and errors, see [How to: Suppress Warnings for Generated Code](../code-quality/how-to-suppress-code-analysis-warnings-for-generated-code.md).  
-
```
Comment (HanqingZheng-ms) [inline redirect-depot/index.md:114]: fffffffffff

**Review by HanqingZheng-ms — CHANGES_REQUESTED** [review]

File: redirect-depot/index.md (hunk 1 above)
Comment (HanqingZheng) [reply redirect-depot/index.md:111]: 头发方法

**Pull request closed without merging**